### ISSUE TITLE
[2020.09] PHP Notices

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#8877]
Please report here any PHP Notice message obtained running the `2020.07` stable or `2020.09-dev` versions.

Follow-up to #8475

**Comment by AlfredSK** [comment]
2020.07 PHP 7.4

```
Trying to access array offset on value of type bool in .../src/Protocol/DFRN.php on line 967
Trying to access array offset on value of type bool in .../src/Protocol/DFRN.php on line 968
```

**Comment by AlfredSK** [comment]
2020.09-dev PHP 7.4

```
Trying to access array offset on value of type bool in .../src/Model/Contact.php on line 241
Trying to access array offset on value of type bool in .../src/Model/Contact.php on line 242
```

**Comment by AlfredSK** [comment]
2020.07 PHP 7.4

```
Trying to access array offset on value of type null in .../src/BaseModule.php on line 143
```

**Comment by AlfredSK** [comment]
2020.07 PHP 7.4

```
Trying to access array offset on value of type null in .../src/Module/Settings/UserExport.php on line 118
```

**Comment by AlfredSK** [comment]
2020.09-dev PHP 7.4

```
Undefined index: baseurl in .../view/theme/frio/theme.php on line 215
```

**Comment by AlfredSK** [comment]
2020.09-dev PHP 7.4
```
Undefined index: cid in .../src/Module/Search/Acl.php on line 353
```

**Comment by annando** [comment]
After pulling the new themes and using one of them:
```PHP Notice:  Undefined variable: nav_bg in /view/theme/frio/php/default.php on line 74```

**Comment by ghost** [comment]
PHP Notice:  Undefined index: addr in /friendica/src/Model/Contact.php on line 1915

Fixed by #9233

**Comment by AlfredSK** [comment]
2020.09-dev:

`Undefined variable: v in .../src/Module/Settings/UserExport.php on line 121`

**Comment by ghost** [comment]
PHP Notice:  Undefined index: addr in /friendica/src/Model/Contact.php on line 1912

Fixed by #9233

**Comment by AlfredSK** [comment]
2020.09-rc

```
Undefined index: uid in .../addon/superblock/superblock.php on line 67
```

Looks like this notice is connected to that fatal error -> [Link](https://github.com/friendica/friendica/issues/8875#issuecomment-689798520)

Fixed by https://github.com/friendica/friendica-addons/pull/1019

**Comment by AlfredSK** [comment]
2020.09-rc

```
Undefined index: id in .../mod/events.php on line 477
Undefined index: cid in .../mod/events.php on line 478
Undefined index: uri in .../mod/events.php on line 479
Undefined index: start in .../mod/events.php on line 485
Undefined index: finish in .../mod/events.php on line 486
Undefined index: adjust in .../mod/events.php on line 490
```
Fixed by #9233

**Comment by AlfredSK** [comment]
Some notices related to UserImport on 2020.09-rc (PHP 7.4):

```
Array to string conversion in .../src/Core/UserImport.php on line 263
Array to string conversion in .../src/Core/UserImport.php on line 265
Trying to access array offset on value of type bool in .../src/Core/UserImport.php on line 288
```

**Comment by AlfredSK** [comment]
2020.09-rc:

`Trying to access array offset on value of type bool in .../src/Model/Mail.php on line 138`

There's also a warning with the same time stamp.

Fixed by #9233

**Comment by AlfredSK** [comment]
2020.09-rc:

`Undefined index: gravity in .../src/Protocol/ActivityPub/Processor.php on line 553`

Fixed by https://github.com/friendica/friendica/pull/9245

**Comment by AlfredSK** [comment]
2020.09-rc:

`Undefined index: text in .../src/Content/PageInfo.php on line 166`

Fixed by https://github.com/friendica/friendica/pull/9245
Finally fixed by https://github.com/friendica/friendica/pull/9248

**Comment by ghost** [comment]
166 ist not fixed

**Comment by MrPetovan** [comment]
Closed by #9252
